hermes-management/src/main/java/pl/allegro/tech/hermes/management/infrastructure/kafka/service/ConsumerGroupsDescriber.java

```diff
@@ -0,0 +1,84 @@
+package pl.allegro.tech.hermes.management.infrastructure.kafka.service;
+
+import org.apache.kafka.clients.admin.AdminClient;
+import org.apache.kafka.clients.admin.ConsumerGroupDescription;
+import org.apache.kafka.clients.admin.MemberDescription;
+import org.apache.kafka.clients.consumer.OffsetAndMetadata;
+import org.apache.kafka.common.TopicPartition;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import pl.allegro.tech.hermes.api.ConsumerGroup;
+import pl.allegro.tech.hermes.api.ConsumerGroupMember;
+import pl.allegro.tech.hermes.api.SubscriptionName;
+import pl.allegro.tech.hermes.common.kafka.ConsumerGroupId;
+import pl.allegro.tech.hermes.common.kafka.KafkaNamesMapper;
+
+import java.util.Collections;
+import java.util.Map;
+import java.util.Optional;
+import java.util.Set;
+import java.util.concurrent.ExecutionException;
+
+import static java.util.stream.Collectors.toSet;
+
+public class ConsumerGroupsDescriber {
+
+    private static final Logger logger = LoggerFactory.getLogger(ConsumerGroupsDescriber.class);
+
+    private final KafkaNamesMapper kafkaNamesMapper;
+    private final AdminClient adminClient;
+    private final LogEndOffsetChecker logEndOffsetChecker;
+
+    public ConsumerGroupsDescriber(KafkaNamesMapper kafkaNamesMapper, AdminClient adminClient, LogEndOffsetChecker logEndOffsetChecker) {
+        this.kafkaNamesMapper = kafkaNamesMapper;
+        this.adminClient = adminClient;
+        this.logEndOffsetChecker = logEndOffsetChecker;
+    }
+
+    public Optional<ConsumerGroup> describeConsumerGroup(SubscriptionName subscription) {
+        ConsumerGroupId consumerGroupId = kafkaNamesMapper.toConsumerGroupId(subscription);
+        try {
+            return describeConsumerGroup(consumerGroupId);
+        } catch (Exception e) {
+            logger.error("Failed to describe group with id: " + consumerGroupId.asString(), e);
```

Comment:
it's always better to avoid string concatenation when logging:
```java
logger.error("Failed to describe group with id: {}", consumerGroupId.asString(), e);
```

Reply:
fixed